sql/mariadb/MariaDBLexer.g4

```diff
@@ -759,6 +759,13 @@ SECOND:                              'SECOND';
 MICROSECOND:                         'MICROSECOND';
 
 
+// userstat plugin Keywords
+
+USER_STATISTICS:                     'USER_STATISTICS';
+CLIENT_STATISTICS:                   'CLIENT_STATISTICS';
+INDEX_STATISTICS:                    'INDEX_STATISTICS';
+TABLE_STATISTICS:                    'TABLE_STATISTICS';
```

Comment:
Please add these new tokens to [keywordsCanBeId](https://github.com/antlr/grammars-v4/blob/master/sql/mariadb/MariaDBParser.g4#L2732).